packages/banner/src/BannerPresenter/presenters.js

```diff
@@ -3,15 +3,12 @@
 import React from "react";
 import cx from "classnames";
 
-import "./banner-presenter.scss";
+import { Icon as BasicIcon, IconButton, Text } from "hig-react";
```

Comment:
These packages are currently being migrated to independent scoped packages as well.